packaging/osx/sharedframework/scripts/postinstall

```diff
@@ -6,8 +6,25 @@
 
 PACKAGE=$1
 INSTALL_DESTINATION=$2
+LIBCRYPTO_NAME=libcrypto.1.0.0.dylib
+LIBSSL_NAME=libssl.1.0.0.dylib
+OPENSSL_PATH=/usr/local/opt/openssl/lib
+
+# Ugly, but we need this to actually get the version
+VERSION=${INSTALL_PKG_SESSION_ID##"com.microsoft.dotnet.sharedframework.Microsoft.NETCore.App."}
+VERSION=${VERSION%%".component.osx.x64"}
+CORE_LIB_FILE_PATH=$INSTALL_DESTINATION/shared/Microsoft.NETCore.App/$VERSION/System.Security.Cryptography.Native.dylib
+
 
 # A temporary fix for the permissions issue(s)
 chmod -R 755 $INSTALL_DESTINATION/shared
 
+if [ -e "$OPENSSL_PATH/$LIBSSL_NAME" ] && [ -e "$OPENSSL_PATH/$LIBCRYPTO_NAME" ]
```

Comment:
if the user hasn't installed openssl/libcrypto should we be failing silently like this? Shouldn't we at least give a warning so the user can go "brew install openssl"?